docs/configuration/configuration.md

```diff
@@ -1302,6 +1306,44 @@ tls_config:
   [ <tls_config> ]
 ```
 
+### `<eureka_sd_config>`
+
+Eureka SD configurations allow retrieving scrape targets using the
+[Eureka](https://github.com/Netflix/eureka) REST API. Prometheus
+will periodically check the REST endpoint for currently running tasks and
+create a target group for every app that has at least one healthy task.
```

Comment:
```suggestion
create a target group for every app that has at least one healthy task.
```

That is not correct prometheus should expose all tasks, healthy can be a meta label.

Reply:
I`m not good at english, so could you help me?

Reply:
I think what he's saying is that the behaviour should be that Prometheus should check this REST endpoint for currently running tasks and create a target group for everything it finds, and whether or not the app/task is 'healthy' should be a metadata label attached to that target. 

So he's commenting that based on your description of the Eureka SD config here in `configuration.md` there's actually an implementation detail in your code that needs to be changed.

Does that make sense @kangwoo?

Reply:
Thank you for your explanation.